pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/juniperkits/JuniperTestSetupUtil.java

```diff
@@ -161,7 +169,20 @@ private static void delete(Connection conn, String tableName, String primaryColu
         }
     }
 
-    public static void deleteJuniperKit(String ddpKitRequestId) {
+    private static void delete(Connection conn, String tableName, String primaryColumn, int id) {
```

Comment:
It looks like this is a copy paste of TestKitUtil.delete().  Since the method is generic, it should go somewhere else, possibly in DBUtil, and then this call and the delete method in TestKitUtil can re-use the same method from there.